src/settingsdialog.ui

```diff
@@ -2016,6 +2016,60 @@ p, li { white-space: pre-wrap; }
                 </property>
                </widget>
               </item>
+              <item>
+               <widget class="QPushButton" name="bsaDateBtn">
+                <property name="toolTip">
+                 <string>
+                          For Skyrim, this can be used instead of Archive Invalidation. It should make AI redundant for all Profiles.
+                          For the other games this is not a sufficient replacement for AI!
+                        </string>
+                </property>
+                <property name="whatsThis">
+                 <string>
+                          For Skyrim, this can be used instead of Archive Invalidation. It should make AI redundant for all Profiles.
```

Comment:
I'm not sure whether this is correct, specifically the fact that this is specifically a Skyrim solution.
I'm not even sure backdating matters for Skyrim, I thought it was used for Oblivion?

Reply:
Git is showing these lines as additions but it isn't, this is code that's been there.

Reply:
Qt designer re-arranged the `.ui` file so this was moved, hence the diff.